site/content/en/reference/pkg/addsubpkg/_index.md

```diff
@@ -0,0 +1,94 @@
+---
+title: "Add Subpkg"
+linkTitle: "Add Subpkg"
+type: docs
+draft: true
+description: >
+   Add a subpackage as a dependency to a Kptfile
+---
+<!--mdtogo:Short
+   Add a subpackage as a dependency to a Kptfile
+-->
+
+`kpt pkg add-subpkg` can be used to add or update Kptfile dependencies
```

Comment:
Make sure nomenclature is a) Well-defined b) Precise c) Consistent.

How are we defining the following terms?

'dependency' vs 'subpackage' vs 'nested package'

Reply:
Got rid of nested package concept as a whole. Defined `subpackages` in https://github.com/GoogleContainerTools/kpt/pull/1034 and pointed to it. Added a short definition to `dependency`